### PR TITLE
Issue 170: Update fonts to match website

### PR DESCRIPTION
**Pull request by dillonfagan**

### CODE REVIEW

File: src/routes/index.svelte [hunk 1]
```diff
@@ -24,7 +24,7 @@
     }
 
     function filterThings(category) {
-        return shownThings.filter(thing => thing.categories.includes(category));
+        return shownThings.filter(thing => thing.categories?.includes(category));
```
Comment (dillonfagan) [inline src/routes/index.svelte:27]: The app was crashing whenever reloading cached data. Adding this null check fixed it.

**Comment by dillonfagan** [comment]
There was an issue with the build pipeline using an older version of node. I upgraded the pipeline to Node 14 and now the build is passing.

**Comment by mdantuono** [comment]
Looks like the checks failed after the last refresh bug push

**Comment by dillonfagan** [comment]
> Looks like the checks failed after the last refresh bug push

The build is passing, but I can't re-run the checks from Github

**Comment by dillonfagan** [comment]
Fonts on the buttons don't look right after fixing the merge conflict. Probably missing a font style. Will fix tonight.

**Pull request merged**